### PR TITLE
Fixes: #2262: Choose correct color for label updates

### PR DESCRIPTION
**Pull request by miketaylr**
r? @zoepage 

r? @karlcow for the milestone color bits.

### CODE REVIEW

**Comment by karlcow** [comment]
also

```
→ nosetests
..................................................................................
----------------------------------------------------------------------
Ran 82 tests in 2.917s

OK
```

File: webcompat/templates/issue/issue-labels-sub.jst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <script type="text/template">
 <% _.each(labels, function(label) { %>
-  <span class="label label-<%= label.attributes[3].value %> js-Label">
+  <span class="label label-<%= label.remoteName %> js-Label">
```
Comment (zoepage) [inline webcompat/templates/issue/issue-labels-sub.jst]: nice!

File: webcompat/templates/web_modules/milestone-editor.jst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <script type="text/template">
   <div class="label-editor-header">
-    <input class="form-field text-field js-MilestoneEditor-search" placeholder="Filter labels">
+    <input class="form-field text-field js-MilestoneEditor-search" placeholder="Filter statuses">
```
Comment (zoepage) [inline webcompat/templates/web_modules/milestone-editor.jst]: 👍

**Pull request merged**